extensions/cpp/language-configuration.json

```diff
@@ -13,7 +13,8 @@
 		{ "open": "{", "close": "}" },
 		{ "open": "(", "close": ")" },
 		{ "open": "'", "close": "'", "notIn": ["string", "comment"] },
-		{ "open": "\"", "close": "\"", "notIn": ["string"] }
+		{ "open": "\"", "close": "\"", "notIn": ["string"] },
+		{ "open": "/*", "close": " */", "notIn": ["string"] }
```

Comment:
Works great, but can you add `"comment"` to the notIn too?

Reply:
Definitely! Thanks for the feedback, I will do this now.